src/types/svg.d.ts

```
@@ -0,0 +1,5 @@
+declare module '*.svg' {
+  import type { FC, SVGProps } from 'react';
```

ESLint

[error] Expected 1 empty line after import statement not followed by another import.

(import/newline-after-import)